src/Microsoft.ML.TimeSeries/SrCnnEntireAnomalyDetector.cs

```diff
@@ -612,9 +627,20 @@ private void SpectralResidual(double[] values, double[][] results, double thresh
 
                     var detres = score > threshold ? 1 : 0;
 
+                    // Anomalies correction by zscore
+                    if (detres > 0)
+                    {
+                        if (_std < _eps || Math.Abs(values[i] - _mean) / _std < _zscoreThreshold)
```

Comment:
nit, if `values[i]` is used in multiple places, we can introduce a variable. Here I feel some parentheses are missing and a bit hard to read? 
`if (_std < _eps || ((Math.Abs(values[i] - _mean) / _std) < _zscoreThreshold))` or slightly better 
```
var zScore = (Math.Abs(values[i] - _mean) / _std;
if (_std < eps || zScore <  _zscoreThreshold)
```

Reply:
updated.

---
In reply to: [553430088](https://github.com/dotnet/machinelearning/pull/5579#discussion_r553430088) [](ancestors = 553430088)